js/MetricQWebView.js

```diff
@@ -58,6 +58,7 @@ class MetricQWebView {
     this.lastThrottledReloadTime = 0
     this.RELOAD_THROTTLING_DELAY = 150
     this.reloadThrottleTimeout = undefined
+    this.relative = false
```

Comment:
`relative` is a relatively ambigous name. I'd suggest a more specific name (but I'm not sure at this point if this variable is actually in the right spot.)